frontend/src/pages/Feedback.js

```diff
@@ -126,20 +151,19 @@ function Feedback() {
         {/* Description */}
         <Form.Group className={styles.form_group}>
           <Form.Label className={styles.form_label}>
-            Feedback/Bug Description
+            Feedback Description
             <span style={{ color: '#ff5e5e' }}>{' *'}</span>
             <br />
             <span className={styles.form_secondary_label}>
-              If you are filing a bug report, please be specific and include
-              steps to reproduce the bug
+              {descriptions[type]}
             </span>
           </Form.Label>
           <StyledInput
```

Comment:
Not related to this PR but the textarea styling is off - no margin and hover state does not change
![image](https://user-images.githubusercontent.com/12566801/99916734-d51e7300-2cd9-11eb-85ba-fad843e3c539.png)